create_auction_result.php

```diff
@@ -1,33 +1,75 @@
-<?php include_once("top_header.php")?>
+<?php
+include_once("top_header.php");
+include_once 'database_connect/connect_db.php';
+$id = $_SESSION['userid'];
+$itemid = $_SESSION['itemid'];
+?>
 
-<div class="container my-5">
 
-<?php
 
-// This function takes the form data and adds the new auction to the database.
+<div class="container my-5">
 
-/* TODO #1: Connect to MySQL database (perhaps by requiring a file that
-            already does this). */
+    <?php
+    $errors = array();
 
+    // This function takes the form data and adds the new auction to the database.
 
-/* TODO #2: Extract form data into variables. Because the form was a 'post'
+    /* TODO #2: Extract form data into variables. Because the form was a 'post'
             form, its data can be accessed via $POST['auctionTitle'], 
             $POST['auctionDetails'], etc. Perform checking on the data to
             make sure it can be inserted into the database. If there is an
             issue, give some semi-helpful feedback to user. */
+    if (isset($_POST['new_auction'])) {
+        $item_name = $_POST['item_name'];
+        $description = $_POST['description'];
+        $condition = $_POST['condition'];
+        $category_ID = $_POST['category_ID'];
+        $starting_price = $_POST['starting_price'];
+        $reserve_price = $_POST['reserve_price'];
+        $start_date = $_POST['start_date'];
+        $start_time = $_POST['start_time'];
+        $end_date = $_POST['end_date'];
+        $end_time = $_POST['end_time'];
+    }
 

```

Comment:
@c20chin  There is one thing to add before inserting new item.

<img width="1388" alt="image" src="https://user-images.githubusercontent.com/115724731/200116476-b4feb1d1-2e38-42cf-bc79-b15712219558.png">

Please see the bidding_status column, it is NULL right now. 

The system should be able to define whether the bidding status is on-going, end, or upcoming. 

To do this, you can compare the input **end_date/time** and **start_date/time** with the **current date**. This function would help automate the real-time status update within our table as well as the display.